src/push-notifications.js

```diff
@@ -206,4 +238,22 @@ function urlBase64ToUInt8Array(base64String) {
   return Uint8Array.from([...rawData].map(char => char.charCodeAt(0)));
 }
 
+/**
+ * Modified from https://stackoverflow.com/questions/4565112
+ */
+function isSupportedBrowser() {
+  const isChromium = window.chrome;
```

Comment:
shouldn't this have the `!== null` here:

```suggestion
  const isChromium = window.chrome !== null;
```